scripts/unpack

```diff
@@ -4,14 +4,14 @@
 # Copyright (C) 2009-2016 Stephan Raue (stephan@openelec.tv)
 # Copyright (C) 2018-present Team LibreELEC (https://libreelec.tv)
 
-. config/options $1
-
 if [ -z "$1" ]; then
   die "usage: $0 package_name"
```

Comment:
die will be unknown until config/functions gets sourced, so

```
echo -e .... >&2
exit 1
```

Same issue `scripts/install` and `scripts/build`.

Reply:
ah yes... very true! will fix.